packages/esm-patient-conditions-app/src/conditions/conditions-form.test.tsx

```diff
@@ -133,22 +133,18 @@ describe('Conditions form', () => {
     const activeStatusInput = screen.getByRole('radio', { name: 'Active' });
     const conditionSearchInput = screen.getByRole('searchbox', { name: /enter condition/i });
 
-    // FIXME: make the date input work
-    const onsetDateInput = screen.getByLabelText('Onset date');
+    const onsetDateInput = screen.getByRole('textbox', { name: /onset date/i });
     expect(onsetDateInput).toBeInTheDocument();
-    // const onsetDateDayInput = within(onsetDateInput).getByRole('spinbutton', { name: /day/i });
-    // const onsetDateMonthInput = within(onsetDateInput).getByRole('spinbutton', { name: /month/i });
-    // const onsetDateYearInput = within(onsetDateInput).getByRole('spinbutton', { name: /year/i });
 
     expect(cancelButton).toBeEnabled();
 
     await user.type(conditionSearchInput, 'Headache');
     await user.click(screen.getByRole('menuitem', { name: /headache/i }));
     await user.click(activeStatusInput);
-    // await user.type(onsetDateInput, '2020-05-05');
-    // await user.type(onsetDateDayInput, '05');
-    // await user.type(onsetDateMonthInput, '05');
-    // await user.type(onsetDateYearInput, '2020');
+    await user.click(onsetDateInput);
+    await user.paste('2020-05-05');
```

Comment:
Unfortunately doing just `user.type` I think triggers a zod schema validation issue and leaves an `invalid date` error